src/lefin/lefin.cpp

```diff
@@ -577,9 +577,10 @@ void lefin::layer( lefiLayer * layer )
 
     dbTechLayerType type( dbTechLayerType::ROUTING );
 
-    if (layer->hasType())
+    if (layer->hasType()) {
```

Comment:
how is this relevant to suppressing swig errors?
I personally don't agree with this formatting style but I think it is best to not randomly introduce changes to the original unless they are universally applied.

Reply:
It was just causing a **Wmisleading-indentation** warning, I reverted it back to the old style and adjusted the indentation instead.